examples/C/hello.c

```diff
@@ -0,0 +1,107 @@
+/*\
+|*| Hello World C example for the Neo6502
+|*| SPDX-License-Identifier: CC0-1.0
+|*|
+|*| This is a very basic C program,
+|*| which sounds a beep and displays some text at a precise screen location.
+|*| Refer to documents/release/api.pdf for more API functions.
+|*|
+|*| To compile and run:
+|*|   $ export CC65_HOME=/usr/share/cc65
+|*|   $ ca65 crt0.s
+|*|   $ ar65 a neo6502.lib crt0.o
```

Comment:
conventionally, this should a .a file in *nix-terms - just noticed it now

Reply:
or maybe an .o file? - im not quite sure what neo6502.lib is exactly
```
$ file examples/C/neo6502.lib 
examples/C/neo6502.lib: xo65 library, version 13
```
if it is truly a static "library" as the .lib extension suggests, then .a is the *nix equvalent sense

Reply:
I was following the porting guide. I think it's all the standard C routines.